app/models/katello/concerns/subscription_aspect_host_extensions.rb

```diff
@@ -5,7 +5,18 @@ module SubscriptionAspectHostExtensions
       include ForemanTasks::Concerns::ActionTriggering
 
       included do
+        SUBSCRIPTION_STATUS_MAP = {
+          :valid => Katello::SubscriptionStatus::FULL_COMPLIANCE,
```

Comment:
Consistency is good - both user facing and source code, if the terminology is 'entitled' then we should use that wherever possible IMO.